### ISSUE TITLE
The new Google Cloud BigQuery connector fails to compile with Akka HTTP 10.2

### ISSUE THREAD

**Issue by ennru** [akka/alpakka#2355]
```
[error] /home/travis/build/akka/alpakka/google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/pagetoken/AddPageToken.scala:39:84: value copy in class HttpRequest cannot be accessed in akka.http.scaladsl.model.HttpRequest
[error]   private def addPageToken(request: HttpRequest, pagingInfo: PagingInfo) = request.copy(
[error]                                                                                    ^
[error] /home/travis/build/akka/alpakka/google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/pagetoken/AddPageToken.scala:40:5: not found: value uri
[error]     uri = request.uri
[error]     ^
[error] /home/travis/build/akka/alpakka/google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/pagetoken/AddPageToken.scala:37:59: value copy in class HttpRequest cannot be accessed in akka.http.scaladsl.model.HttpRequest
[error]   private def convertRequestToGet(req: HttpRequest) = req.copy(method = HttpMethods.GET, entity = HttpEntity.Empty)
[error]                                                           ^
[error] /home/travis/build/akka/alpakka/google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/pagetoken/AddPageToken.scala:37:64: not found: value method
[error]   private def convertRequestToGet(req: HttpRequest) = req.copy(method = HttpMethods.GET, entity = HttpEntity.Empty)
[error]                                                                ^
[error] /home/travis/build/akka/alpakka/google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/pagetoken/AddPageToken.scala:37:90: not found: value entity
[error]   private def convertRequestToGet(req: HttpRequest) = req.copy(method = HttpMethods.GET, entity = HttpEntity.Empty)
[error]                                                                                          ^
[error] /home/travis/build/akka/alpakka/google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/pagetoken/AddPageToken.scala:35:12: value copy in class HttpRequest cannot be accessed in akka.http.scaladsl.model.HttpRequest
[error]     getReq.copy(uri = getReq.uri.withPath(getReq.uri.path ?/ path))
[error]            ^
[error] /home/travis/build/akka/alpakka/google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/pagetoken/AddPageToken.scala:35:17: not found: value uri
[error]     getReq.copy(uri = getReq.uri.withPath(getReq.uri.path ?/ path))
[error]                 ^
```

https://travis-ci.com/github/akka/alpakka/jobs/357276569#L1105

Refs https://github.com/akka/akka-http/issues/3290

**Comment by raboof** [comment]
Since https://github.com/akka/akka-http/pull/2938#discussion_r380169261 we want users to use `withXxx` methods rather than the `copy` method. I guess we could backtrack on that but perhaps we should just mention it in the migration guide.

**Comment by ennru** [comment]
Fixed with #2356
